report-pr-errors

```diff
@@ -314,6 +317,27 @@ class Logs(object):
                                    .format(self.all_logs[-1], err)]
         self.preview_error_log = '\n'.join(error_log_lines).strip()
 
+    def get_versions(self):
+        try:
+            (alibuild_version, status) = getout("alibuild version")
+            if status:
+                self.alibuild_version = "Could not get alibuild version"
+            else:
+                self.alibuild_version = alibuild_version
+        except Exception as e:
+            print("Error while getting alibuild version:", e, file=sys.stderr)
+            self.alibuild_version = "Could not get alibuild version"
+        try:
+            (alidist_version, status) = getout("git -C ./alidist rev-parse --short HEAD")
```

Comment:
Do not assume alidist is using git.

Reply:
It now fails when alidist is not a git repo. I'm not sure how to get the alidist version when it doesn't have history though

```bash
$ git --git-dir ./alidist/.git rev-parse --short HEAD
f0f86990
$ mv ./alidist/.git ./alidist/.git2
$ git --git-dir ./alidist/.git rev-parse --short HEAD
fatal: not a git repository: './alidist/.git'
```

﻿<img width="300" alt="Screenshot 2024-06-26 at 14 33 47" src="https://github.com/alisw/ali-bot/assets/47090312/1fb30063-7faf-421a-a453-346d3b12b967"> <img width="300" alt="image" src="https://github.com/alisw/ali-bot/assets/47090312/1e560ff8-5795-414d-9f6b-ba167ca77b14">

Reply:
Why not simply passing the alibuild version (obtained via `aliBuild version`) and the alidist one as parameters / environment variables? This way you do not need to know how to get them from the script reporting about the builds. You could actually reuse ALIBUILD_SLUG and ALIDIST_SLUG which I think we already define.

Reply:
Are they also set for CI jobs? I've only found the definition for the Jenkins jobs.

(if they aren't we can also add them before merging here and just use those)

Reply:
Implemented getting the version from the env vars, as discussed